opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/BaseApi.java

```diff
@@ -472,6 +472,17 @@ public void setStringProvider(StringProvider stringProvider) {
         this.stringProvider = stringProvider;
     }
 
+    protected String buildHttpGetParams(Map<String, String> params,
+                                        String encoding) throws UnsupportedEncodingException {
```

Comment:
Good idea, I changed that in 65f8dce11d45bdf6341e77240860f932fc42de74 and construct the query using Apache's `URIBuilder`.